isb_web/main.py

```diff
@@ -252,6 +254,24 @@ async def login():
 """
 
 
+@app.get("/thingpage/{identifier:path}", include_in_schema=False)
+async def get_thing_page(request: fastapi.Request, identifier: str, session: Session = Depends(get_session)) -> templates.TemplateResponse:
+    # Retrieve record from the database
+    item = sqlmodel_database.get_thing_with_id(session, identifier)
+    if item is None:
+        raise fastapi.HTTPException(
+            status_code=404, detail=f"Thing not found: {identifier}"
+        )
+    content = await thing_resolved_content(identifier, item)
+    content_str = json.dumps(content)
+    return templates.TemplateResponse(
+        "thing.html", {
+            "request": request,
+            "thing_json": content_str
```

Comment:
This `thing_json` needs to match up with the string we render in the html template.